### PR TITLE
feat:support apply llmaz to any ns

### LINKED ISSUE #141: Support install llmaz at any namespace
**What would you like to be added**:

Right now, because we hardcode the namespace of llmaz-system, we can't support this.

 https://github.com/InftyAI/llmaz/blob/f02361f23f7f40df8ca5bff4510ce5a1a37bdff7/pkg/cert/cert.go#L25

**Why is this needed**:

**Completion requirements**:

This enhancement requires the following artifacts:

- [ ] Design doc
- [ ] API change
- [ ] Docs update

The artifacts should be linked in subsequent comments.

### PR DESCRIPTION
**Pull request by qinguoyi**
#### What this PR does / why we need it
Support install llmaz at any namespace

#### Which issue(s) this PR fixes
Fixes #141 

#### Special notes for your reviewer
i update the cert ns from hardcode to chart configuration, but there also has many "llamz-system" hardcode, such as :
https://github.com/InftyAI/llmaz/blob/4c35e017d7f78600a5a5624571850374fd9a4b46/Makefile#L300-L303
https://github.com/InftyAI/llmaz/blob/4c35e017d7f78600a5a5624571850374fd9a4b46/docs/installation.md?plain=1#L10-L16
https://github.com/InftyAI/llmaz/blob/4c35e017d7f78600a5a5624571850374fd9a4b46/docs/installation.md?plain=1#L44-L49
https://github.com/InftyAI/llmaz/blob/4c35e017d7f78600a5a5624571850374fd9a4b46/config/default/kustomization.yaml#L1-L3
https://github.com/InftyAI/llmaz/blob/4c35e017d7f78600a5a5624571850374fd9a4b46/chart/crds/openmodel-crd.yaml#L7-L17

what can i do for these?
#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

**Comment by kerthcet** [comment]
Thanks @qinguoyi for the help, let me take a try first.

File: chart/templates/deployment.yaml [hunk 1]
```diff
@@ -71,8 +71,8 @@ spec:
           10 }}
         securityContext: {{- toYaml .Values.controllerManager.kubeRbacProxy.containerSecurityContext
           | nindent 10 }}
-      securityContext:
-        runAsNonRoot: true
+      securityContext: {{- toYaml .Values.controllerManager.podSecurityContext | nindent
```
Comment (kerthcet) [inline chart/templates/deployment.yaml:74]: Is this change related? If not, we can have another PR to cover your case.

File: cmd/main.go [hunk 2]
```diff
@@ -58,11 +63,9 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.StringVar(&secretNamespace, "webhook-secret-namespace", "llmaz-system", "The namespace of the secret that contains the webhook server's TLS certificate and key.")
```
Comment (kerthcet) [inline cmd/main.go:68]: Let's rename this to `namespace` intuitively. Same to other places, secret is just one of the resources in the namespace I think.

File: cmd/main.go [hunk 3]
```diff
@@ -58,11 +63,9 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
```
Comment (kerthcet) [inline cmd/main.go:63]: Please don't, let's make sure this variables only available in main function for now.

File: cmd/main.go (hunk 2 above)
Comment (qinguoyi) [reply cmd/main.go:68]: ok, i update this soon.

File: cmd/main.go (hunk 3 above)
Comment (qinguoyi) [reply cmd/main.go:63]: ok ,i revert this soon.

**Comment by qinguoyi** [comment]
as suggested, i re-push the code. @kerthcet

**Comment by kerthcet** [comment]
Thanks @qinguoyi this makes sense to me. But we have some confusions on the document right now because we have two different ways to install lws, one with kustomize, another with helm chart, I'll update the document later to make sure people can participate in the project development easier. Thanks anyway.

/lgtm
/approve
/kind feature
